core/lib/libtomcrypt/sub.mk

```diff
@@ -114,6 +114,7 @@ _CFG_CRYPTO_WITH_ASN1 := $(call cryp-one-enabled, RSA DSA ECC)
 _CFG_CRYPTO_WITH_FORTUNA_PRNG := $(call cryp-all-enabled, AES SHA256)
 
 cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -DLTC_SMALL_CODE -DLTC_NO_FAST
+cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -Os
```

Comment:
If `-Os` does not belong here, I would suggest to simply drop `CFG_CRYPTO_SIZE_OPTIMIZATION` and ensure `-Os` is passed to all core and core-libraries (which is not the case in LTC right now).

What do you think?